pkg/controller/catalogsourceconfig/catalogsourceconfig_controller.go

```diff
@@ -91,6 +91,7 @@ type ReconcileCatalogSourceConfig struct {
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileCatalogSourceConfig) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	log.Printf("Reconciling CatalogSourceConfig %s/%s\n", request.Namespace, request.Name)
+	log.Warning("DEPRECATION NOTICE: The CatalogSourceConfig API is deprecated in future versions. Please visit this link for futher details: https://docs.openshift.com/container-platform/4.5/release_notes/ocp-4-5-release-notes.html#ocp-4-5-marketplace-apis-deprecated")
```

Comment:
Same comment as the other, I think we don't want to throw warnings for defaults since we will migrate them cleanly without admin intervention

Reply:
@kevinrizza do you mean we don't want to throw warnings for default Opsrcs? My understanding was that we want warnings for ALL CSCs.

Reply:
@adellape can you verify that this would be the correct header?

Reply:
Discussed with Anik/Kevin and decided it should be a 4.4 target: https://docs.openshift.com/container-platform/4.4/release_notes/ocp-4-4-release-notes.html#ocp-4-4-marketplace-apis-deprecated

Reply:
@shawn-hurley updated the PR with the 4.4 release note target.